static/templates/bot_avatar_row.hbs

```diff
@@ -21,6 +21,9 @@
             {{/if}}
         </div>
     </div>
+    <div id="edit_bot_modal" class="modal modal-bg hide fade" tabindex="-1" role="dialog"
+      aria-labelledby="change_full_name_modal_label" aria-hidden="true">
+    </div>
```

Comment:
This modal is scoped wrong -- there's one copy per bot, which is a problem both because they have the same ID, and also because you'll try to open/close several modal if you have several bots!  I think you want to leave this in e.g. `settings_overlay.html`.

Reply:
ok sure